vmb/matrimony/admin.py

```diff
@@ -291,9 +292,25 @@ def has_add_permission(self, request, obj=None):
         return False
 
 
+class MatrimonyProfileForm(forms.ModelForm):
+    class Meta:
```

Comment:
We will need to extend the `__init__` method of the form to mark `birth_place` and `current_place` fields as **required**. You can do something similar to this for the fields in question here:

https://github.com/jpsoffice/vmb/blob/master/vmb/matrimony/forms.py#L138

Reply:
I have made the changes as required. Thank you for the reference.